### DISCUSSION TITLE
Access camera feed

### DISCUSSION THREAD

**Discussion by useless21** [flet-dev/flet#3135] [Q&A]
How do i access phone camera feed in real time in flet window... Cv2 seems to always open its own window which does have a feed but I want it to show on flet window

**Comment by FeodorFitsner** [comment]
`Camera` control is still in development: https://github.com/flet-dev/flet/issues/1281
and to support your scenario I guess we need 1) "streaming" support for Camera control and 2) "streaming" support for `Video` control.

How do you like to see "streaming", btw, Python stream, file, method giving chunks of bytes, etc?

**Comment by RRozi** [comment]
I also work with Flet and Cv2. I just change the value of the Image widget frame by frame

**Comment by usermicrodevices** [comment]
there is also a [libcamera](https://libcamera.org/). it has direct access to hardware, python binding and support for multiple platforms. is it possible to make this available in flet?
